ansible/ansible-ai-connect-service
# one-click trial landing page

Jira Issue: <https://issues.redhat.com/browse/AAP-26231>

## Description

Landing page for the 90 days Trial.

![image](https://github.com/ansible/ansible-ai-connect-service/assets/49379/9f37781c-faa5-47aa-8074-d40d2d80f22c)

## Testing

- Set `ANSIBLE_AI_ENABLE_ONE_CLICK_TRIAL` True
- Connect to the service with OIDC user, the Org should not have any key set
- You should be able to accept the T&C and start a Trial period

No API permission changes should be triggered at this stage. Another PR will come for that.

## Production deployment
